docs/auth.md

```diff
@@ -160,13 +160,7 @@ This section describes how to configure the following authentication schemes for
 
 This section describes how to configure a `basic-auth` type `Secret` for use with Git. In the example below,
 before executing any `Steps` in the `Run`, Tekton creates a `~/.gitconfig` file containing the credentials
-specified in the `Secret`. When the `Steps` execute, Tekton uses those credentials to retrieve
-`PipelineResources` specified in the `Run`.
-
-> :warning: **`PipelineResources` are [deprecated](deprecations.md#deprecation-table).**
->
-> Consider using replacement features instead. Read more in [documentation](migrating-v1alpha1-to-v1beta1.md#replacing-pipelineresources-with-tasks)
-> and [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md).
+specified in the `Secret`. 
```

Comment:
I'm a bit confused about this section of the docs and it would be good if someone more familiar can review whether these docs are relevant without pipelineresources-- maybe @vdemeester?